mkosi

```diff
@@ -401,9 +441,11 @@ def btrfs_subvol_delete(path: str) -> None:
     # Delete the subvolume now that all its descendants have been deleted
     run(["btrfs", "subvol", "delete", path], stdout=DEVNULL, stderr=DEVNULL, check=True)
 
-def btrfs_subvol_make_ro(path: str, b:bool=True) -> None:
+
+def btrfs_subvol_make_ro(path: str, b: bool = True) -> None:
```

Comment:
I may be wrong but last time I looked at this the assignment of default values was supposed to not use whitespace around it.

Reply:
Whitespace is supposed to be there iff the parameter has a type specified, see 554a455934c9214ffc4aa5011aec399a2967a424.